src/Microsoft.SqlTools.ServiceLayer/SqlContext/QueryExecutionSettings.cs

```diff
@@ -26,6 +29,14 @@ public class QueryExecutionSettings
         /// </summary>
         private const int DefaultMaxXmlCharsToStore = 2097152; // 2 MB - QE default
 
+        /// <summary>
+        /// Default selection of returning an actual XML showplan with all batches
+        /// </summary>
+        private ExecutionPlanOptions DefaultExecutionPlanOptions = new ExecutionPlanOptions() { 
+            IncludeActualExecutionPlanXml = false,
```

Comment:
>false [](start = 44, length = 5)

I think false is the default value for a bool so you probably don't need the object initialization block in this case IIRC